definitions/webfontloader_v1.x.x/flow_>=v0.23.x/webfontloader_v1.x.x.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+declare module 'webfontloader' {
+  declare type WebFontConfig = {
+    loading?: () => any,
```

Comment:
Can we change these `any`s to `mixed` instead?

This will ensure that if this type ever leaks out, we don't have to worry about type information getting lost